keyboards/work_louder/work_board/keymaps/default/keymap.c

```diff
@@ -16,26 +16,23 @@
 
 #include QMK_KEYBOARD_H
 
-enum planck_layers {
+enum layers {
   _QWERTY,
   _LOWER,
   _RAISE,
   _ADJUST
 };
 
 enum tap_dances {
-    ENC_TAP,
+    ENC_TAP
```

Comment:
Comma should stay, as it prevents issues with clang-format. 
```suggestion
    ENC_TAP,
```